packaging/hudi-hadoop-mr-bundle/pom.xml

```diff
@@ -67,9 +67,6 @@
                   <include>org.apache.hudi:hudi-common</include>
                   <include>org.apache.hudi:hudi-hadoop-mr</include>
 
-                  <!-- TODO(HUDI-3239) remove this -->
-                  <include>org.scala-lang:scala-library</include>
-
```

Comment:
Should the following dep be removed in this pom as well?  Similar for the other bundle poms, `hudi-hadop-mr/pom.xml`,`hudi-common/pom.xml` where scala dep is added in #4531.

```
    <!-- Scala -->
    <!-- TODO(HUDI-3239) remove this dep -->
    <dependency>
      <groupId>org.scala-lang</groupId>
      <artifactId>scala-library</artifactId>
      <version>${scala.version}</version>
    </dependency>
```

Reply:
Oh, great catch! 
Did mindless search and delete somehow bundles didn't bubble up in it. Sorry about that.